docs/topics/queryable-encryption.rst

```diff
@@ -0,0 +1,158 @@
+        EmbeddedModelField,
+        EncryptedCharField,
+        EncryptedEmbeddedModelField,
+    )
+
+
+    class Patient(models.Model):
+        patient_name = models.CharField(max_length=255)
+        patient_id = models.BigIntegerField()
+        patient_record = EmbeddedModelField("PatientRecord")
+
+        def __str__(self):
+            return f"{self.patient_name} ({self.patient_id})"
+
+
+    class PatientRecord(EmbeddedModel):
+        ssn = EncryptedCharField(max_length=11)
+        billing = EncryptedEmbeddedModelField("Billing")
+        bill_amount = models.DecimalField(max_digits=10, decimal_places=2)
+
+
+    class Billing(EmbeddedModel):
+        cc_type = models.CharField(max_length=50)
+        cc_number = models.CharField(max_length=20)
+
+.. _qe-migrations:
+
+Migrations
+----------
+
+Once you have defined your models, create migrations with:
+
+.. code-block:: console
+
+    $ python manage.py makemigrations
+
+Then run the migrations with:
+
+.. code-block:: console
+
+    $ python manage.py migrate --database encrypted
+
+Now create and manipulate instances of the data just like any other Django
+model data. The fields will automatically handle encryption and decryption,
+ensuring that :ref:`sensitive data is stored securely in the database
+<manual:qe-features-encryption-at-rest>`.
+
+Routers
+-------
+
+The example above requires a :ref:`database router
+<qe-configuring-database-routers-setting>` to direct operations on models with
+encrypted fields to the appropriate database. It also requires the use of a
+:ref:`router for embedded models <configuring-database-routers-setting>`. Here
+is an example that includes both:
+
+.. code-block:: python
+
+    # myproject/settings.py
+    DATABASE_ROUTERS = [
+        "django_mongodb_backend.routers.MongoRouter",
+        "myproject.routers.EncryptedRouter",
+    ]
+
+
+Querying encrypted fields
+-------------------------
+
+In order to query encrypted fields, you must define the queryable encryption
+query type in the model field definition. For example, if you want to query the
+``ssn`` field for equality, you can define it as follows:
+
+.. code-block:: python
+
+    class PatientRecord(EmbeddedModel):
+        ssn = EncryptedCharField(max_length=11, queries={"queryType": "equality"})
+        billing = EncryptedEmbeddedModelField("Billing")
+        bill_amount = models.DecimalField(max_digits=10, decimal_places=2)
+
+Then you can perform a query like this:
+
+.. code-block:: console
+
+    >>> patient = Patient.objects.get(patient_record__ssn="123-45-6789")
+    >>> patient.name
+    'John Doe'
+
+.. _qe-available-query-types:
+
+Available query types
+~~~~~~~~~~~~~~~~~~~~~
+
+The ``queries`` option should be a dictionary that specifies the type of queries
+that can be performed on the field. Of the :ref:`available query types
+<manual:qe-fundamentals-encrypt-query>` Django MongoDB Backend currently
+supports:
+
+- ``equality``
+- ``range``
```

Comment:
There's nothing baked into Django MongoDB Backend that says it can only support equality and range. rangePreview should work just as well on MongoDB 7, or the prefix, suffix, and substring queries available in public preview in MongoDB 8.2. Right?